vendor/autoload.php

```diff
@@ -2,6 +2,24 @@
 
 // autoload.php @generated by Composer
 
+if (PHP_VERSION_ID < 50600) {
```

Comment:
@andryes Should we be adding these files in "vendor" folder or should it be ignored?